src/components/EditMessageForm/EditMessageForm.js

```diff
@@ -150,38 +144,20 @@ class EditMessageForm extends React.Component {
         <ImageDropzone
           accept={this.props.acceptedFiles}
           multiple={this.props.multipleUploads}
-          disabled={
-            this.props.numberOfUploads >= this.props.maxNumberOfFiles
-              ? true
-              : false
-          }
+          disabled={this.props.numberOfUploads >= this.props.maxNumberOfFiles}
           handleFiles={this.props.uploadNewFiles}
         >
           <form onSubmit={this.props.handleSubmit}>
             {this.renderEmojiPicker()}
             {this.renderUploads()}
-            {this.props.emojiPickerIsOpen && (
```

Comment:
No style side-effects because of the order those components are rendered?

Reply:
I double checked: doesn't look like it, the emoji picker is always positioned outside of the rest of the EditMessageForm, the order in which it's rendered doesn't change anything here. (it uses absolute positioning with a z-index)